### DISCUSSION TITLE
Vulnerability to zbar [CVE-2023-40889] discovered

### DISCUSSION THREAD

**Discussion by kazuohprc** [mchehab/zbar#264] [General]
Is there any information on the compatible versions?

**Comment by pr-apes** [comment]
Sorry, @mchehab, this might be worth to take a look at.

https://cve.report/CVE-2023-40889 rates the vulnerability as critical (9.8 out of 10).

Many thanks for your help.

**Comment by pr-apes** [comment]
@kazuohprc,

right after commenting on your general discussion, I saw #263.

Since it is a reported issue older than this, it might be worth to move all discussion there.

Many thanks for your help.
